tests/testutils.go

```diff
@@ -125,6 +177,21 @@ func GetInt64AttributeValue[T any](item T, attributeName string) int64 {
 	return f.Int()
 }
 
+func GetRandomSubset(values []int64, subsetSize int) []int64 {
+	copyValues := make([]int64, len(values))
+	copy(copyValues, values)
+	rand.Seed(time.Now().UnixNano())
```

Comment:
is it useful to log the seed to potentially reproduce something/debug?

Reply:
I can add that.

Reply:
done in last commit.